bitmap-ram-rod/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const bitmapper = require('./model/bitmap.js');
+const transform = require('./lib/transformer.js');
+
+let transformations = Object.keys(transform);
+let fileName = 'palette-bitmap';
+
+console.log(bitmapper);
+
+for(let i = 1; i < transformations.length; i++) {
+  bitmapper(`./assets/${fileName}.bmp`, transform[transformations[i]], `${fileName}-${transformations[i]}`);
```

Comment:
This should be bitmap.renderImage() I believe as that allows it to create the files and bitmapper is just an object with properties not a function.